src/Microsoft.IdentityModel.Tokens/RsaSecurityKey.cs

```diff
@@ -31,10 +31,17 @@
 
 namespace Microsoft.IdentityModel.Tokens
 {
+    /// <summary>
+    /// Represents a Rsa security key.
+    /// </summary>
     public class RsaSecurityKey : AsymmetricSecurityKey
     {
         private bool? _hasPrivateKey;
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RsaSecurityKey"/> class.
+        /// </summary>
+        /// <param name="rsaParameters"><see cref="RSAParameters"/></param>
```

Comment:
@brentschmaltz Why is the capitalization "Rsa" for the security key and "RSA" for parameters?